<h1 style='font-size:40px'> A Multi-Container App with Kubernetes</h1>

<h2 style='font-size:30px'> 
    The Path to Production
</h2>
<div> 
    <ul style='font-size:20px'> 
        <li>
            Esse capítulo do curso será dedicado à implantação do projeto desenvolvido na seção 8 para um cluster Kubernetes.
        </li>
    </ul>
 </div>

<figure>
        <center style='margin-top:20px'>
            <img src='img/14_project_diagram.png'>
                <figcaption>  Diagrama do projeto.</figcaption>
        </center>
    </figure>

<h2 style='font-size:30px'> 
    Recreating the Deployment
</h2>
<div> 
    <ul style='font-size:20px'> 
        <li>
            Aula em que configuramos o .yaml do Deployment dos Pods com as imagens "multi-client".
        </li>
    </ul>
 </div>

In [2]:
! git add .
! git commit -m 'Pesquisar sobre o argumento "replicas" dos .yaml\'s de Deployments'
! git push

[master c399805] Aula 210
 44 files changed, 736 insertions(+), 54 deletions(-)
 create mode 100644 projects/complex-gh/.DS_Store
 create mode 100644 projects/complex-gh/client/.gitignore
 create mode 100644 projects/complex-gh/client/Dockerfile
 create mode 100644 projects/complex-gh/client/Dockerfile.dev
 create mode 100644 projects/complex-gh/client/README.md
 create mode 100644 projects/complex-gh/client/nginx/default.conf
 create mode 100644 projects/complex-gh/client/package.json
 create mode 100644 projects/complex-gh/client/public/favicon.ico
 create mode 100644 projects/complex-gh/client/public/index.html
 create mode 100644 projects/complex-gh/client/public/logo192.png
 create mode 100644 projects/complex-gh/client/public/logo512.png
 create mode 100644 projects/complex-gh/client/public/manifest.json
 create mode 100644 projects/complex-gh/client/public/robots.txt
 create mode 100644 projects/complex-gh/client/src/App.css
 create mode 100644 projects/complex-gh/client/src/App

<p style='color:red'> Vi Aula 210; Pesquisar sobre o argumento "replicas" dos .yaml's de Deployments; Quantos Pods são produzidos quando declaramos mais de um container?; Aula 211</p>